---
title: ''
layout: archive
permalink: /publications/
author_profile: yes
---

{% include base_path %}

For a full list of publications, please visit my **[google scholar profile](https://scholar.google.com/citations?user=8QLuIWgAAAAJ&hl)**.

List of selected publications:
* SE Johnstone<sup>\*</sup>, **A Reyes<sup>\*</sup>**, ..., MJ Aryee<sup>+</sup> and BE Bernstein<sup>+</sup>. Large-Scale Topological Changes Restrain Malignant Progression in Colorectal Cancer. *Cell*, 2020. doi:
[10.1016/j.cell.2020.07.030](https://doi.org/10.1016/j.cell.2020.07.030).
* J Chavez<sup>\*</sup>, C Barberena-Jonas<sup>\*</sup>, JE Sotelo-Fonseca<sup>\*</sup>, ..., L Collado-Torres<sup>+</sup> and **A Reyes<sup>+</sup>**. Programmatic access to bacterial regulatory networks with regutools. *Bioinformatics*, 2020. doi:
[10.1093/bioinformatics/btaa575](https://doi.org/10.1093/bioinformatics/btaa575).
* P Kimes<sup>+</sup> and **A Reyes<sup>+</sup>**. Reproducible and replicable comparisons using SummarizedBenchmark. *Bioinformatics*, 2018. doi:
[10.1093/bioinformatics/bty627](https://doi.org/10.1093/bioinformatics/bty627).
* **A Reyes<sup>+</sup>** and W Huber<sup>+</sup>. Alternative start and termination sites of transcription drive most transcript isoform differences across human tissues. *Nucleic Acids Research*, 2017. doi: [10.1093/nar/gkx1165](https://doi.org/10.1093/nar/gkx1165).
* P Brennecke<sup>\*</sup>, **A Reyes<sup>\*</sup>**, S Pinto<sup>\*</sup>, K Rattay<sup>\*</sup>, ..., and LM Steinmetz. Single-cell transcriptome analysis reveals coordinated ectopic gene-expression patterns in medullary thymic epithelial cells. *Nature Immunology*, 2015. doi: [10.1038/ni.3246](https://doi.org/10.1038/ni.3246).
* D Klimmeck<sup>\*</sup>, N Cabezas-Wallscheid<sup>\*</sup>, **A Reyes<sup>\*</sup>**, ..., and A Trumpp. Transcriptome-wide profiling and posttranscrip- tional analysis of hematopoietic stem/progenitor cell differentiation toward myeloid commitment. *Stem Cell Reports*, 2014. doi: [10.1016/j.stemcr.2014.08.012](https://doi.org/10.1016/j.stemcr.2014.08.012).
* N Cabezas-Wallscheid<sup>\*</sup>, D Klimmeck<sup>\*</sup>, J Hansson<sup>\*</sup>, DB Lipka<sup>\*</sup>, **A Reyes<sup>\*</sup>**, ..., and A Trumpp. Identification of regulatory networks in HSCs and their immediate progeny via integrated proteome, transcriptome, and DNA methylome analysis. *Cell Stem Cell*, 2014. doi: [10.1016/j.stem.2014.07.005](https://doi.org/10.1016/j.stem.2014.07.005).
*  **A Reyes<sup>\*</sup>**, S Anders<sup>\*</sup>, ..., and W Huber. Drift and conservation of differential exon usage across tissues in primate species. *PNAS*, 2013. doi: [10.1073/pnas.1307202110](https://doi.org/10.1073/pnas.1307202110).
* S Anders<sup>\*</sup>, **A Reyes<sup>\*</sup>** and W Huber. Detecting differential usage of exons from RNA-seq data. *Genome Research*, 2012. doi: [10.1101/gr.133744.111](https://doi.org/10.1101/gr.133744.111). 

<sup>*</sup> Denotes co-first authors

<sup>+</sup> Denotes co-corresponding authors
