%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% "ModernCV" CV and Cover Letter
% LaTeX Template
% Version 1.11 (19/6/14)
%
% This template has been downloaded from:
% http://www.LaTeXTemplates.com
%
% Original author:
% Xavier Danaux (xdanaux@gmail.com)
%
% License:
% CC BY-NC-SA 3.0 (http://creativecommons.org/licenses/by-nc-sa/3.0/)
%
% Important note:
% This template requires the moderncv.cls and .sty files to be in the same 
% directory as this .tex file. These files provide the resume style and themes 
% used for structuring the document.
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%----------------------------------------------------------------------------------------
%	PACKAGES AND OTHER DOCUMENT CONFIGURATIONS
%----------------------------------------------------------------------------------------

\documentclass[11pt,a4paper,sans]{moderncv} % Font sizes: 10, 11, or 12; paper sizes: a4paper, letterpaper, a5paper, legalpaper, executivepaper or landscape; font families: sans or roman

\moderncvstyle{classic} % CV theme - options include: 'casual' (default), 'classic', 'oldstyle' and 'banking'
\moderncvcolor{black} % CV color - options include: 'blue' (default), 'orange', 'green', 'red', 'purple', 'grey' and 'black'

\usepackage{lipsum} % Used for inserting dummy 'Lorem ipsum' text into the template
\usepackage{amsmath}
\usepackage[english]{babel}
\usepackage{enumitem}
\usepackage{lastpage}
\pagestyle{fancy}
\rfoot{\thepage\ of \pageref{LastPage}}
\fancyfoot[RE,LO]{Alejandro Reyes, PhD}
\renewcommand{\footrulewidth}{0.4pt}

\usepackage[scale=.85]{geometry} % Reduce document margins
\setlength{\hintscolumnwidth}{2.5cm} % Uncomment to change the width of the dates column
%\setitemize{label=\textbullet}
\setitemize{label=\raisebox{.5\height}{\scalebox{0.8}{\textbullet}}}
\setlength{\itemsep}{0.1pt}
\setlength{\parskip}{0pt}
\setlength{\parsep}{0pt}
%\setlength{\makecvtitlenamewidth}{10cm} % For the 'classic' style, uncomment to adjust the width of the space allocated to your name

%----------------------------------------------------------------------------------------
%	NAME AND CONTACT INFORMATION SECTION
%----------------------------------------------------------------------------------------

\firstname{Alejandro} % Your first name
\familyname{Reyes} % Your last name

% All information in this block is optional, comment out any lines you don't need
\title{\emph{Curriculum Vitae (\today)}}
%\address{2440 Massachusetts Avenue, Apt \#12}{Cambridge, 02140, USA}
%\mobile{+49 - 15772345859}
%\mobile{+1 - 617 961 2609}
%\fax{(000) 111 1113}
%
\extrainfo{
  Birth date: February 12, 1989 \\
  \href{http://orcid.org/0000-0001-8717-6612}{ORCID: 0000-0001-8717-66}\\
  \href{https://scholar.google.com/citations?user=8QLuIWgAAAAJ}{Google Scholar: 8QLuIWgAAAAJ}\\
  \href{https://publons.com/a/389744/}{Publons: 389744}\\
  \href{https://twitter.com/areyesq}{Twitter: @areyesq}
}
\email{alejandro.reyes.ds@gmail.com}

%\email{\date{}}
\homepage{http://alejandroreyes.org}{}
% The first argument is the url for the clickable link, the second argument is the url displayed in the template - this allows special characters to be displayed such as the tilde in this example
%\extrainfo{Birth date: February 12, 1989.}
\photo[70pt][0.4pt]{reyes_pic.jpg} % The first bracket is the picture height, the second is the thickness of the frame around the picture (0pt for no frame)
%\quote{"A witty and playful quotation" - John Smith}


%----------------------------------------------------------------------------------------
\begin{document}
\makecvtitle % Print the CV title
\vspace{-1.2cm}
%----------------------------------------------------------------------------------------
%	EDUCATION SECTION
%----------------------------------------------------------------------------------------
\section{Summary}
I develop computational strategies that enable
the translation of large amounts of data into biological knowledge. 
%Broadly, I am interested in (1) understanding processes by which
%transcript isoforms contribute to cellular phenotypes and disease
%conditions and (2) integrating multi-omic data to unravel the
%molecular phenotypes in cancer. 
To ensure reproducibility of results and effective
dissemination of code, I implement documented workflows, software 
packages and graphic interfaces.

\section{Education}

\cventry{09/2011 - 10/2015}{Ph.D. in Bioinformatics}{\emph{European Molecular
    Biology Laboratory (EMBL) and Ruprecht Karl University of Heidelberg}}{Germany}{}{\emph{Summa cum laude.}}
\cventry{08/2007 - 06/2011}{B.Sc. in Genomic Sciences}{\emph{National
    Autonomous University of Mexico (UNAM)}}{Cuernavaca, Mexico}{}{With honours.}

%----------------------------------------------------------------------------------------
%	WORK EXPERIENCE SECTION
%----------------------------------------------------------------------------------------

\section{Work Experience}
\cventry{04/2020 - present}{Principal Scientist I}{}{Novartis Institutes for BioMedical Research}{Basel, Switzerland}{}
\cventry{11/2016 - 03/2020}{Postdoctoral Research Fellow}{}{Dana-Farber Cancer Institute and Harvard T.H. Chan School of Public Health}{Boston, USA}{
  \begin{itemize}
    \item Advisor: Prof. Rafael Irizarry.
    \item Project 1 (ongoing): I am investigating how the epigenome and 3D structure of the
      genome is altered in colorectal cancer through the analysis and integration
      of multi-omic datasets. 
    \item Designed an assay to identify the presence
      of circulating tumor DNA in the bloodstream based on the sensitive
      detection of allelic frecuency imbalances caused by large
      genomic copy number alterations; Co-authored three manuscripts, including one
       as corresponding author; Co-authored an
       R/Bioconductor package.
  \end{itemize}
}
\cventry{10/2015 - 09/2016}{Bridging Postdoctoral Fellow}{}{EMBL}{Heidelberg, Germany}{
  \begin{itemize}
  \item Advisor: Dr. Wolfgang Huber.
  \item Analyzed data from the Genotype-Tissue Expression project to study transcript isoform dynamics across human tissues; Published a first-author article.
  \end{itemize}
}
\cventry{09/2011 - 10/2015}{PhD Student}{}{EMBL}{Heidelberg, Germany}{
  \begin{itemize}
  \item Advisor: Dr. Wolfgang Huber.
  \item Developed statistical software to analyze RNA-seq data; Used public datasets to investigate transcript
isoform dynamics across tissues and species; Collaborated with
experimentalists, participated in the design of research questions and experiments, and led the
computational analysis of interdisciplinary projects; Published 9
scientific articles, of which 6 as first author; Authored 3
R/Bioconductor packages; Trained PhD students and postdocs during yearly courses and workshops.
  \end{itemize}
}
\cventry{08/2010 - 06/2011}{Research Trainee}{}{EMBL}{Heidelberg, Germany}{
  \begin{itemize}
  \item Advisor: Dr. Wolfgang Huber.
  \item Developed statistical methods to identify changes in
    transcript isoform regulation between different biological conditions.
  \end{itemize}
}
\cventry{06/2010 - 08/2010}{Research Trainee}{}{Weizmann Institute of Science}{Rehovot, Israel}{
  \begin{itemize}
  \item Advisor: Prof. Doron Lancet.
  \item Developed a computational pipeline to identify human genetic variants affecting olfactory receptors; Co-authored a peer-reviewed article.
  \end{itemize} }
\cventry{11/2009 -- 06/2010}{Undergraduate Research Assistant}{}{UNAM}{Cuernavaca, Mexico}{
  \begin{itemize}
  \item Advisors: Prof. Julio Collado-Vides and Prof. Enrique Morett.
  \item Evaluated methods to map transcription start sites in \emph{E. coli} using high-throughput sequencing data.
   \end{itemize}
}
  
\section{Scientific publications}
\vspace{-.1cm}
\footnotesize{$^{\ast}$ Contributed equally.} \hspace{.5cm}
\footnotesize{$^{\dagger}$ Shared last authorship.} \\
\\
\vspace{-.1cm}
\textbf{Submitted for peer review:}
\begin{itemize}
\item SE Johnstone$^{\ast}$, \textbf{\underline{A Reyes}}$^{\ast}$,
  ..., M Aryee$^{\dagger}$ and BE Bernstein$^{\dagger}$. A survey of
  genome topology in colorectal tumors reveals large-scale
  compartmental changes that restrain malignant progression. (Under
  review)
\item Y Qi, \textbf{\underline{A Reyes}}, ..., B Zhang. Data-driven
  polymer model for mechanistic exploration of diploid genome
  organization. \textit{biorXiv},
  2020. \href{https://doi.org/10.1101/2020.02.27.968735}{doi: 10.1101/2020.02.27.968735}
\end{itemize}
\vspace{.1cm}
\textbf{Selected publications:}
\begin{itemize}
\item P Kimes$^{\ast}$ and \textbf{\underline{A Reyes}}$^{\ast}$. Reproducible and replicable comparisons using SummarizedBenchmark. \textit{Bioinformatics}, 2018. \href{https://doi.org/10.1093/bioinformatics/bty627}{doi: 10.1093/bioinformatics/bty627}
\item \textbf{\underline{A Reyes}}$^{\dagger}$ and W Huber$^{\dagger}$. Alternative start and termination sites of transcription drive most transcript isoform differences across human tissues. \textit{Nucleic Acids Research}, 2017. \href{https://doi.org/10.1093/nar/gkx1165}{doi: 10.1093/nar/gkx1165}
\item P Brennecke$^{\ast}$, \textbf{\underline{A Reyes}}$^{\ast}$, S Pinto$^{\ast}$, K Rattay$^{\ast}$, ..., W Huber$^{\dagger}$, B Kyewski$^{\dagger}$ and LM Steinmetz$^{\dagger}$. Single-cell transcriptome analysis reveals coordinated ectopic gene-expression patterns in medullary thymic epithelial cells. \textit{Nature Immunology}, 2015. \href{https://doi.org/10.1038/ni.3246}{doi: 10.1038/ni.3246}
\item D Klimmeck$^{\ast}$, N Cabezas-Wallscheid$^{\ast}$, \textbf{\underline{A Reyes}}$^{\ast}$, ..., W Huber$^{\dagger}$ and A Trumpp$^{\dagger}$. Transcriptome-wide profiling and posttranscriptional analysis of hematopoietic stem/progenitor cell differentiation toward myeloid commitment. \textit{Stem Cell Reports}, 2014. \href{https://doi.org/10.1016/j.stemcr.2014.08.012}{doi: 10.1016/j.stemcr.2014.08.012}
\item N Cabezas-Wallscheid$^{\ast}$, D Klimmeck$^{\ast}$, J Hansson$^{\ast}$, DB Lipka$^{\ast}$, \textbf{\underline{A Reyes}}$^{\ast}$, ..., W Huber$^{\dagger}$, MD Milsom$^{\dagger}$, C Plass$^{\dagger}$, J Krijgsveld$^{\dagger}$ and A Trumpp$^{\dagger}$. Identification of regulatory networks in HSCs and their immediate progeny via integrated proteome, transcriptome, and DNA methylome analysis. \textit{Cell Stem Cell}, 2014. \href{https://doi.org/10.1016/j.stem.2014.07.005}{doi: 10.1016/j.stem.2014.07.005}
\item \textbf{\underline{A Reyes}}$^{\ast}$, S Anders$^{\ast}$, ..., W Huber. Drift and conservation of differential exon usage across tissues in primate species. \textit{PNAS}, 2013. \href{https://doi.org/10.1073/pnas.1307202110}{doi: 10.1073/pnas.1307202110}
\item S Anders$^{\ast}$, \textbf{\underline{A Reyes}}$^{\ast}$ and W Huber. Detecting differential usage of exons from RNA-seq data. \textit{Genome Research}, 2012. \href{https://doi.org/10.1101/gr.133744.111}{doi: 10.1101/gr.133744.111}
\end{itemize}
\vspace{.1cm}
\textbf{Other publications:}
\vspace{.1cm}
\begin{itemize}
\item K Korthauer$^{\ast}$, P Kimes$^{\ast}$, ... ,
  \textbf{\underline{A Reyes}}, ..., SC Hicks. A practical guide to
  methods controlling false discoveries in computational
  biology. \textit{Genome Biology}, 2019. \href{https://doi.org/10.1186/s13059-019-1716-1}{doi: 10.1186/s13059-019-1716-1}
\item M Ruiz-Velasco, ..., \textbf{\underline{A Reyes}}, ..., JB Zaugg. CTCF-mediated chromatin loops between promoter and gene body regulate alternative splicing across individuals. \textit{Cell Systems}, 2017. \href{http://dx.doi.org/10.1016/j.cels.2017.10.018}{doi: 10.1016/j.cels.2017.10.018}
\item MM Parker, ..., \textbf{\underline{A Reyes}}, ..., PJ Casaldi. RNA sequencing identifies novel non-coding RNA and exon-specific effects associated with cigarette smoking. \textit{BMC Medical Genomics}, 2017. \href{https://doi.org/10.1186/s12920-017-0295-9}{doi: 10.1186/s12920-017-0295-9}
\item R Scognamiglio, ..., \textbf{\underline{A Reyes}}, ..., A Trumpp. Myc depletion induces a pluripotent dormant state mimicking diapause. \textit{Cell}, 2016. \href{https://doi.org/10.1016/j.cell.2015.12.033}{doi: 10.1016/j.cell.2015.12.033}
\item W Huber, ..., \textbf{\underline{A Reyes}}, ..., M Morgan. Orchestrating high-throughput genomic analysis with Bioconductor. \textit{Nature Methods}, 2015. \href{https://doi.org/10.1038/ni.3246}{doi: 10.1038/ni.3246}
\item \textbf{\underline{A Reyes}}, ..., W Huber. Mutated SF3B1 is associated with transcript isoform changes of the genes UQCC and RPL31 both in CLLs and uveal melanomas. \textit{bioRxiv}, 2013. \href{https://doi.org/10.1101/000992}{doi: 10.1101/000992}
\item K Zarnack$^{\ast}$, J K\"{o}nig$^{\ast}$, ..., \textbf{\underline{A Reyes}}, ..., NM Luscombe$^{\dagger}$ and J Ule$^{\dagger}$. Direct competition between hnRNP C and U2AF65 protects the transcriptome from the uncontrolled exonization of Alu elements. \textit{Cell}, 2013. \href{https://doi.org/10.1016/j.cell.2012.12.023}{doi: 10.1016/j.cell.2012.12.023}
\item T Olender, ..., \textbf{\underline{A Reyes}}, ..., D Lancet. Personal receptor repertoires: olfaction as a model. \textit{BMC Genomics}, 2012. \href{https://doi.org/10.1186/1471-2164-13-414}{doi: 10.1186/1471-2164-13-414}
\end{itemize}
\vspace{-.13cm}

\section{Software development}
\vspace{-.1cm}
\begin{itemize}
\item \href{http://www.bioconductor.org/packages/release/bioc/html/DEXSeq.html}{DEXSeq}: Inference of differential exon usage from RNA-seq data. \textit{R/Bioconductor}.
\item \href{http://bioconductor.org/packages/release/data/experiment/html/pasilla.html}{pasilla}: Package with count data of a pasilla knock-down RNA-seq experiment. \textit{R/Bioconductor}.
\item \href{http://bioconductor.org/packages/release/data/experiment/html/Single.mTEC.Transcriptomes.html}{Single.mTEC.Transcriptomes}: Transcriptome data and analysis of mouse mTECs. \textit{R/Bioconductor}.
\item \href{http://www.bioconductor.org/packages/release/bioc/html/SummarizedBenchmark.html}{SummarizedBenchmark}: Inference of differential exon usage from RNA-seq data. \textit{R/Bioconductor}.
\end{itemize}
\vspace{-.2cm}

\section{Honors}
\begin{itemize}
\item Mexican National System of Researchers (SNI I): becoming and
  remaining a member of the SNI requires demonstration of significant contributions in
  research and teaching.\\
\end{itemize}

\section{Presentations and Posters}
\vspace{-.1cm}
\textbf{Invited talks}
\begin{itemize}
\item Models, Inference and Algorithms Seminar. Broad Institute. Cambridge, USA, 2019.
\item Alnylam Genomics Club. Alnylam Pharmaceuticals Inc. Cambridge, USA, 2018.
\item LIIGH Seminar. International Laboratory for Human Research.  Juriquilla, Mexico, 2018.
\item Blue Seminar. EMBL, Heidelberg, Germany, 2017. 
\item Evolution of Biological Traits. Center for Advanced Studies (LMU), Munich, Germany, 2017.
\item Seminarios de Investigaci\'on. Universidad del Valle de Atemajac, Queretaro, Mexico, 2017.
\item Genomeeting workshop. National Institute of Genomic Medicine, Mexico City, Mexico, 2016. 
\item 15th Annual BCI-McGill Workshop. Bellairs Research Institute, Holetown, Barbados, 2016.
\item C1omics Workshop. Manchester Cancer Research Centre, Manchester, UK, 2015.
\item Interpretation of Next Generation Sequencing Data
  Workshop. University of Heidelberg, Germany, 2015. 
\item RADIANT General Meeting. Telethon Institute of Genetics and Medicine, Pozzuoli, Italy, 2015.
\item ``Manejo Inteligente de Datos e Informaci\'on''. Mexican Institute of Transportation, Queretaro, Mexico, 2014.
\item European Conference on Computational Biology \emph{RADIANT} Workshop. Strasbourg, France, 2014.
\item Statistical Analysis of RNA-seq Data. Pasteur Institute, Paris, France, 2013.
\item BioC Conference. Fred Hutchison Cancer Research Center, Seattle, USA, 2013.
\end{itemize}
\vspace{.1cm}
\textbf{Abstracts selected for a talk}
\begin{itemize}
\item The Biology of Genomes. Cold Spring Harbor Laboratory, Cold Spring Harbor, USA, 2014. 
\end{itemize}
\vspace{.1cm}
\textbf{Poster presentations}
\begin{itemize}
\item Single-cell Genomics Conference. Hubrecht Institute, Utrecht, Netherlands, 2015.
\item Cancer Genomics Conference. EMBL, Heidelberg, Germany, 2012.
\end{itemize}

%------------------------------------------------
\vspace{-.2cm}
\section{Teaching}
\vspace{-.1cm}
\textbf{Organizer}
\begin{itemize}
\item ``Building tidy tools'' workshop. UNAM, Cuernavaca, Mexico, 2019.
\item Latin American BioC Developers Workshop. UNAM, Cuernavaca, Mexico, 2018.
\end{itemize}
\vspace{.1cm}
\textbf{Mentor and lecturer}
\begin{itemize}
\item Detecting differentially expressed genes with RNA-seq Data, Dana-Farber Cancer Institute. Boston, USA, 2019.
\item Productivity tools in Unix, Dana-Farber Cancer Institute. Boston, USA, 2019. 
\item Workshop on Transcriptomics, Harvard University. Cambridge, USA, 2017. 
\item UNAM's II Summer School in Bioinformatics. UNAM,  Juriquilla, Mexico, 2017. 
\item Replicathon2017: Consistency of Large Pharmacogenomic Studies. University of Puerto Rico Río Piedras, Puerto Rico, 2017.
\item Statistics and Computing in Genome Data Science. University of
  Padua, Bressanone, Italy, 2015.
\item Data Analysis for Genome Biology. University of Padua, Bressanone, Italy, 2014.
\item Computational Statistics for Genome Biology. University of
  Padua, Bressanone, Italy, 2013.
\item BioC Conference. Fred Hutchison Cancer Research Center, Seattle, USA, 2012.
\end{itemize}
\vspace{.1cm}
\textbf{Teaching assistant}
\begin{itemize}
\item Introduction to Data Science: BST260. Harvard T.H. Chan School of Public Health, Boston, USA, 2017. 
\item Advanced topics in Evolutionary Genomics. {\v C}zern\'y  Krumov, Czech Republic, 2013.
\item Computational Statistics for Genome Biology. University of Padua,
  Bressanone, Italy, 2012. 
\item Computational Statistics for Genome Biology.University of Padua,
  Bressanone, Italy, 2011. 
\item Introduction to R/Bioinformatics. UNAM, Cuernavaca, Mexico, 2010.
\end{itemize}

\section{Skills}
\cvitem{Programming languages}{R/Bioconductor, python, perl, C, \LaTeX, mySQL}
\cvitem{Languages}{Spanish (native), English (advanced), Italian (advanced), German (basic)}
\cvitem{Other}{Piano}

%\section{References}
%\begin{itemize}
%\item Dr. Wolfgang Huber (whuber@embl.de).
%Principal Investigator and Senior Scientist at the EMBL.
%\item Prof. Lars M. Steinmetz (larsms@stanford.edu).
%Professor of Genetics at Stanford University. Co-Director of the Stanford Genome Technology Center.
%Principal Investigator, Associate Head of Genome Biology and Senior Scientist at the EMBL.
%\end{itemize}

\end{document}
